style.css

```diff
@@ -0,0 +1,64 @@
+.container {
+  display: flex;
+  flex-direction: row;
+  padding: 25px;
+}
+
+.logo {
+  flex-grow: 1;
+  flex-shrink: 5;
+}
+
+.content {
+  display: flex;
+  flex-direction: column;
+  padding: 25px;
+  background-image: url("./images/background.svg");
+  background-size: 100vw;
+  background-repeat: no-repeat;
+}
+
+h1 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  align-items: center;
+  color: #172b4d;
+  margin: 12px 0;
+  padding-top: 50px;
+}
+
+p {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 24px;
+  color: #344563;
+  margin: 12px 0;
+}
+
+h2 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  color: #7f8cff;
+  margin: 0 8px;
+}
```

Comment:
- [ ] Kindly change the font for this `h2` to make it like the Figma design and the color and remove the underline.

| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/35535130/154376865-74ecad34-f6ea-43d9-8252-206c3cd17dce.png) | ![image](https://user-images.githubusercontent.com/35535130/154376819-433f6bf9-500e-4a66-867e-3590dee85967.png) |